src/engine/QueryExecutionContext.h

```diff
@@ -229,6 +237,8 @@ class QueryExecutionContext {
   // `std::nullopt`, the result is not cached.
   std::optional<PinResultWithName> pinResultWithName_ = std::nullopt;
 
+  MaterializedViewsManager* materializedViewsManager_ = nullptr;
```

Comment:
leave it uninitialized, the constructor initializes, but then we at least get warnings, should we ever forget it.